### ISSUE TITLE
RFE Underwater movement for Support Vehicles with environmental sealing.

### ISSUE THREAD

**Issue by Gribbly1** [MegaMek/megamek#3821]
Support vehicles with the environmental sealing chassis modification should be able to enter depth 1+ water in the same way as Mechs, driving along the bottom - at risk of breaches as normal.

Please could you take a look at the viability of implementing this in Megamek?

I raised a rules question on the CGI forum regarding the equivalent Combat Vehicle chassis modification, based upon the existence of the Rommel and Ajax 'Sealed' variants.

**Comment by Gribbly1** [comment]
https://bg.battletech.com/forums/tactical-operations/environmental-sealing-and-underwater-movement-re-sealed-ajax-and-rommel/

Both combat vehicles and support vehicles with sealed chassis can move underwater.

**Comment by Gribbly1** [comment]
Non-naval superheavies should also be able to enter depth 1 water without chassis modification, p190 Tac Ops AU&E.

**Comment by Gribbly1** [comment]
I'd be grateful if someone could run their eyes over this one to give some indication of how difficult it would be to implement and if it's currently on any radar, cheers.

**Comment by Gribbly1** [comment]
This is working perfectly in yesterday's nightly. Many thanks for looking at this and resolving it so quickly.


~~I noticed that the movement costs vary with water depth according to the normal table, as expected. From a rules point of view that difference is likely intended to represent the difference between 2 level tall mechs partially or fully submerged.~~

~~Vehicles are half as tall as mechs and are fully submerged in depth 1 water. I would possibly expect them to pay either the +1 or +3 movement cost for all of their submerged movement - I can see an argument for either way. I'll put in a forum question.~~

<img width="497" height="95" alt="Image" src="https://github.com/user-attachments/assets/5eae85a0-02da-4762-a061-eccdf4a4a0d7" />
 
<img width="291" height="264" alt="Image" src="https://github.com/user-attachments/assets/02b752e9-2762-4498-b4c7-2fba9dc48960" />

<img width="333" height="296" alt="Image" src="https://github.com/user-attachments/assets/3487888e-2c45-4bb1-8d88-fea5589e74f6" />

**Comment by Gribbly1** [comment]
Looking at this I think the MP costs might need adjusting.

<img width="512" height="501" alt="Image" src="https://github.com/user-attachments/assets/af5dd2ce-aee3-4f53-9374-aa8490526978" />

It should be +3 MP for all underwater hexes for vehicles. MM is currently doing +1 for depth 1, and +2 for depth 2 or more.

<img width="313" height="459" alt="Image" src="https://github.com/user-attachments/assets/93c8cded-1f54-4abd-8162-ccdab1c750c5" />

**Comment by Gribbly1** [comment]
Please reopen this for visibility until the MP costs are corrected.

**Comment by IllianiBird** [comment]
I reopened this, but closed it again as I think it would be better for a fresh report to be opened. This was likely never addressed because the report was closed following the addressing of the initial issue.

If I reopened this report it would just remain buried with other 4 month old reports.
